### PR TITLE
Add Instructions for Hosting the Site 

### PR DESCRIPTION
**Pull request by JoeDailey**
Currently, setting up the site for public use is undocumented. This is a risk given that I'm the only hoster. If this changes, or if the site otherwise needs to be migrated, this will be helpful.

---

These files were taking from the current site.

### CODE REVIEW

File: server/update.sh [hunk 1]
```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+# This scrip pulls and builds the latest version of the site.
+# It can be run manually, but it is much more useful as a cron job.
+#
+# For example, this crontab entry will build the site every 15 minutes.
+#
+#   0 */15 * ? * * sh /home/admin/delta.khk.org/server/update.sh /home/admin/delta.khk.org /var/www/delta.khk.org
+
+if [[ $# -eq 0 ]] ; then
```
Comment (celowsky) [inline server/update.sh]: Should there be exactly one argument? If so, a better check might be:
```bash
if [[ $# != 1 ]]; then
```

Comment (JoeDailey) [reply server/update.sh]: There should be exactly two arguments. I'm not super familiar with bash. What would be a good way to check this?

Comment (celowsky) [reply server/update.sh]: You could do something like:
```bash
if [[ 2 != $# ]]; then
    printf '\n%s\n' "[$(date)]Jekyll Build Failed: No path to Jekyll site supplied. Please provide two arguments. \n"
fi
```

**Comment by JoeDailey** [comment]
We no longer need this because we are hosting on Github Pages. I have referenced this potential solution in the README in case anybody attempts to host this manually in the future.

**Pull request closed without merging**